python/dicebeard/__init__.py

```diff
@@ -97,16 +102,18 @@ def __init__(self, *args, **kwargs):
             return
 
         # TODO removed image use while we fix a bug with image processing.
-        # i = no_of_dice
-        # out, total = self.my_dice.train(i)
-        # await self.sender.sendPhoto(out)
+        i = no_of_dice
+        out, total = self.my_dice.train(i)
+        await self.sender.sendPhoto(out)
         # TODO replace this dice hacking with something better like pydice
-        dice = [random.randint(1,6) for x in range(no_of_dice)]
-        total = sum(dice)
-        await self.sender.sendMessage(str(dice))
-
+        #dice = [random.randint(1,6) for x in range(no_of_dice)]
+        #total = sum(dice)
+        #await self.sender.sendMessage(str(dice))
+        
+        my_listener = self.bot.create_listener() 
+        my_listener.capture([{'from':{'id':msg['from']['id']}}])
```

Comment:
This also needs to catch the chat id as well as the user id (otherwise it will also answer requests from private chats with the individual).